test/testunusedvar.cpp

```diff
@@ -4033,7 +4085,7 @@ class TestUnusedVar : public TestFixture {
                               "    std::cout << \"test\" << std::endl;\n"
                               "    delete fred;\n"
                               "}");
-        ASSERT_EQUALS("", errout.str());
+        ASSERT_EQUALS("[test.cpp:4]: (style) Variable 'fred' is allocated memory that is never used.\n", errout.str());
```

Comment:
:+1:   not sure why we asserted that no error should be detected. It looks good to warn here. A `TODO_ASSERT_EQUALS` would have been better.